### ISSUE TITLE
Relative date formatting never returns the "previous" value

### ISSUE THREAD

**Issue by gshahbazian** [malcommac/SwiftDate#646]
Run this in the SwiftDate.playground:
```swift
let date = Date() - 1.years
let relative = date.toRelative()
relative == "last year"
```

Expected that since the long and longConvenient formats define the previous year text that should be returned. (If I am misunderstanding how to use get a format that will give the previous string value, please let me know).

Actual `relative` = `a year ago`

Tested on v5.1.0

**Comment by Valpertui** [comment]
I've fixed this in : https://github.com/malcommac/SwiftDate/issues/646

**Comment by malcommac** [comment]
Thank you for your PR, just merged for 5.2.0
